lib/tasks/occupation.rake

```diff
@@ -1,6 +1,39 @@
 require "csv"
+require "roo"
 
 namespace :occupation do
+  task rapids_codes_scraper: :environment do
+    if Date.current.wday.eql?(0) || ENV["FORCE"] == "true"
+      puts "Importing RAPIDS codes"
+      xlsx = Roo::Spreadsheet.open("https://www.apprenticeship.gov/sites/default/files/wps/apprenticeship-occupations.xlsx")
+      xlsx.sheet(0).parse(headers: true).each do |row|
+        occupation = Occupation.find_or_initialize_by(name: row["RAPIDS TITLE"])
+        hybrid_start_hours = nil
+        hybrid_end_hours = nil
+        hybrid_hours = nil
+
+        if row["HYBRID"]
+          hybrid_start_hours = row["HYBRID"].match(/(\d{1,}) -/)
+          hybrid_end_hours = row["HYBRID"].match(/- (\d{1,})/)
+        end
+
+        if hybrid_start_hours && hybrid_end_hours
+          hybrid_hours = (hybrid_start_hours[1]..hybrid_end_hours[1])
+        end
```

Comment:
@jenlu33 thanks for making the range change. Do you know if the Hybrid hours will always be a range? Otherwise we need to account for the case if it is just a single number, and set that for the start and end range.

Reply:
@littleforest Sorry it took me a while to look back at this again. I tried googling, but I couldn't find any real answers. I added a range that just has the single number for both the starting and end numbers... but it seems like not the best solution?

Reply:
Yes, that's exactly right! I think this can be simplified a bit by putting all of this under the initial check for `if row["HYBRID"]` (and maybe give the `row["HYBRID"]` a variable name):

```
        if row["HYBRID"]
          hours = row["HYBRID"]
          hybrid_start_hours = hours.match(/(\d{1,})\s*-/)
          hybrid_end_hours = hours.match(/-\s*(\d{1,})/)

          if hybrid_start_hours && hybrid_end_hours
            hybrid_hours = (hybrid_start_hours[1]..hybrid_end_hours[1])
          else
            hybrid_hours = (hours..hours)
          end
        end
```

Also I added a `\s*` in the regex to have a little more flexibility in how the numbers come through (so it will match on "2000-4000" and "2000 - 4000".